### PR TITLE
Add dbmap and SSL info to the box

### PR DESCRIPTION
**Pull request by rssk**
# problem statement
- the 'box doesnt have some config, namely the dbmap and ssl info for coins

# solution
- `Put it in the box`

### CODE REVIEW

File: packages/coins-deposit-box/test.js [hunk 1]
```diff
@@ -16,6 +16,13 @@ assert(isURL(depositBox.apiURL), 'API URL');
 assert(isString(depositBox.cookieName), 'Auth cookie name');
 assert(isString(depositBox.env), 'Environment');
 assert(isURL(depositBox.webURL), 'Web URL');
+assert(isString(depositBox.dbmap), '/coins/config/dbmap.json');
```
Comment (swashcap) [inline packages/coins-deposit-box/test.js]: `assert` [doesn’t compare strings](https://nodejs.org/api/assert.html#assert_assert_value_message). Use `assert.equal`.

Also, comparing equality is a brittle test. Why not ensure the filename has a `.json` extension and call it good?

**Comment by swashcap** [comment]
Mind updating [steelpenny’s dbmap.json ref?](https://github.com/MRN-Code/quarterback/blob/master/packages/steelpenny/config/default.json#L19).

File: packages/coins-deposit-box/test.js (hunk 1 above)
Comment (rssk) [inline packages/coins-deposit-box/test.js]: 👍

**Comment by swashcap** [comment]
:+1: Looks great! Data Exchange tests aren't passing due to hard-coded values. Whatever.

**Pull request merged**